### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by EleanorRoberts**
Reviewing the main portfolio page

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	
+</head>
+<body>
+	<!-- navbar -->
+<nav>
```
Comment (jesshamby) [inline portfolio.html]: Perhaps add indentations to nav etc. as they're children of body? :)

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+		<div class="project-item">
+			<img class="project-image" src="Nova1.jpeg" alt="black and white Nova" />
+			<div class="project-info">
+				<h3>Project Three</h3>
+				<div class="project-description">This is the description of the third project</div>
```
Comment (jesshamby) [inline portfolio.html]: Could this be a p tag so it is more semantic?

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+				<div>
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+	</section>
+</div>
+	<!-- about me -->
+<div>
```
Comment (jesshamby) [inline portfolio.html]: Guessing this div is added for formatting reason however if you remove it so you just have the section could you then format the section instead? Center it etc. :)

File: portfolio.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	<a href="#about" class="nav-item">About Me</a>
+	<a href="#contact" class="nav-item">Contact</a>
+	<a href="Eleanor Roberts CV.pdf" class="nav-item cv" target="_blank">CV</a>
+</nav>
+	<!-- header -->
+<header id="home">
+	<h1>Eleanor Roberts</h1>
+</header>
+	<!-- projects -->
+<div>
```
Comment (Roomsycakes01) [inline portfolio.html]: Unnecessary div

File: portfolio.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+		<h2>About Me</h2>
+		<div class="about">
+			<p>Hi, I'm Ellie and I like coding, my dog Nova and gaming. I'm currently completing iO Academy's Full Stack Track programming bootcamp (and loving it!).</p> 
+			<p>In my free time, I enjoy learning Japanese, walking my dog, playing piano, spending far too much time on Zelda and finding new hobbies.</p>
+			<p>If you'd like to know more about my professional history, please visit my <a href="timeline.html" target="_blank">timeline!</a></p>
+		</div>
+	</section>
+</div>
+	<!-- contact -->
+<div>
```
Comment (jesshamby) [inline portfolio.html]: Same comment as line 70- think you could possibly remove this div :)

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+			<a href="https://www.linkedin.com/in/eleanor-roberts-80117a16a/" class="fa fa-linkedin"></a>
+			<a href="https://www.freecodecamp.org/elmon" class="fa fa-free-code-camp"></a>
+			<a href="mailto: eleanorlroberts@googlemail.com" class="fa fa-envelope-square"></a>
+			<a href="https://github.com/EleanorRoberts" class="fa fa-github-square"></a>
+		</div>
+	</section>
+</div>
+	<!-- footer -->
+<footer>
+	<section>
```
Comment (Roomsycakes01) [inline portfolio.html]: Could just use a footer here with margin auto to get rid of the section and avoid the header warning.

File: portfoliostyle.css [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+}
+
+h3 {
+	color: black;
+	border-radius: 15px;
+	padding: 5px 0;
+}
+
+p a {
+	color: white;;
```
Comment (jesshamby) [inline portfoliostyle.css]: double ;

File: portfoliostyle.css [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+	background-position: top center;
+}
+
+/*most element styles*/
+
+nav {
+	position: fixed;
+	top: 1vh;
+	right: 1vw;
+	background-color: grey;
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: Change to Hex or RGB colours to avoid eternal shame

File: portfoliostyle.css [hunk 9]
```diff
@@ -0,0 +1,391 @@
...
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	color: white;
+}
+
+h1 {
+	font-size: 100px;
+	margin: 0;
+	text-align: left;
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: I don't think this is needed

File: timeline.html [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+	<a href="#timeline" class="nav-item">Timeline</a>
+	<a href="#skills" class="nav-item">Skills</a>
+	<a href="Eleanor Roberts CV.pdf" class="nav-item cv" target="_blank">CV</a>
+</nav>
+	<!-- header -->
+<header id="home">
+	<h1>History of Me</h1>
+</header>
+	<!-- timeline -->
+<div>
```
Comment (jesshamby) [inline timeline.html]: Is this div necessary with the section after it? :)

File: timeline.html [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+	<section id="timeline">
+		<div class="timeline">
```
Comment (jesshamby) [inline timeline.html]: Could you get rid of this div and give the section above a class of timeline?

File: portfoliostyle.css [hunk 12]
```diff
@@ -0,0 +1,391 @@
...
+  width: 0;
+  z-index: 1;
+  right: 15px;
+  border: medium solid white;
+  border-width: 8px 0 8px 25px;
+  border-color: transparent transparent transparent grey;
+}
+
+.right::before {
+  content: " ";
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: if you're trying to add in a space write the line:
white-space:pre; 
after the content

File: timeline.html [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+		  <div class="container left">
+		    <div class="content">
```
Comment (jesshamby) [inline timeline.html]: Think you can perhaps get rid of this div also so it is just in the container left div and format it all in there?

File: timeline.html [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+		      <h3>Risk Management Operations Specialist</h3>
+			  <p class="timeline-company">Celgene/BMS</p>
+			  <p class="timeline-date">May 2019 - October 2020</p>
+		      <p>In my role as (Data Entry) RMO Specialist, I did many things with my time which were all very impressive and involved lots of PV which I absolutely love and was not the reason I then transferred out of the industry and into coding which is much more suied to my tastes. I love logic puzzles, and whilst the overly large excel spreadsheets at Celgene/BMS were satisfying to correctly create/adjust/filter they are not as fun as what is essentially a big beautiful new rubix cube for me to play with.</p>
+		    </div>
+		  </div>
+		  <div class="container right">
+		    <div class="content">
```
Comment (jesshamby) [inline timeline.html]: Same as above: can maybe get rid of this div? :)

File: timeline.html [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	
+</head>
+<body>
+	<!-- navbar -->
+<nav>
```
Comment (jesshamby) [inline timeline.html]: Perhaps indent nav etc. as they're the children of the body tag :)

**Review by jesshamby — CHANGES_REQUESTED** [review]
This looks awesome and all the pics of nova just make it! Some pretty impressive hovers and links :) 

Think the main comments I made were questioning whether some div's were necessary. Very few comments for the css. 

Love it! :)

File: portfoliostyle.css (hunk 9 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Specifically the text-align left or the whole section?

File: portfoliostyle.css [hunk 16]
```diff
@@ -0,0 +1,438 @@
...
+	content: "Hover over the below roles to learn more!";
+	color: #FFFFFF;
+	background-color: #000000;
+	opacity: 0.5;
+	padding: 5px;
+	border-radius: 5px;
+}
+
+.timeline::after {
+	content: "";
```
Comment (willthornhill) [inline portfoliostyle.css:220]: This content property seems unnecessary?

File: portfoliostyle.css [hunk 17]
```diff
@@ -0,0 +1,438 @@
...
+}
+
+.right::after {
+	left: -16px;
+}
+
+.content {
+	padding: 20px 30px;
+	background-color: #808080
+;
```
Comment (willthornhill) [inline portfoliostyle.css]: Semi-colon could be inline with the background-color for consistency

**Review by willthornhill — COMMENTED** [review]
I really like your site and couldn't spot many issues with your code.

I like how to use /*notes*/ to organise your code.

One way to improve would be to add more pictures of Nova, currently not enough.

File: timeline.html [hunk 18]
```diff
@@ -0,0 +1,80 @@
...
+				<li>Coding</li>
+			</ul>
+		</div>
+	</section>
+	<!-- footer -->
+	<footer>
+		<div>NovaInc. Copyright 2021</div>
+	</footer>
+</body>
+</html>
```
Comment (Roomsycakes01) [inline timeline.html]: Extra line here king

File: timeline.html [hunk 19]
```diff
@@ -0,0 +1,80 @@
...
+			</div>
+			<!-- right container -->
+			<div class="container right">
+				<div class="content">
+					<h3>Bartending</h3>
+					<p class="timeline-company">The Lord Nelson
+					</p>
+					<p class="timeline-date">September 2016 - May 2019
+					</p>
+					<p class="to-hide">Working as a bartender at this local pub, I had a great time making friends with the regulars and seeing them get spectacularly drunk (and occassionally helping them get home afterwards). I worked on my beautiful social skills, ability to let drunken bulls**t roll off my back, and general stock management/time management/interpersonal and management skill improvement.
```
Comment (Roomsycakes01) [inline timeline.html]: Text overflows box on iphone 5 at management/interpersonal section.

File: portfoliostyle.css (hunk 16 above)
Comment (EleanorRoberts) [reply portfoliostyle.css:220]: Hey! You can only use ::after with content, even if that content is nothing :)

File: portfoliostyle.css (hunk 17 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Deleted :)

File: timeline.html (hunk 19 above)
Comment (EleanorRoberts) [reply timeline.html]: Edited the text for less run along. Will have to revisit depending on actual content 🧐

File: timeline.html (hunk 18 above)
Comment (EleanorRoberts) [reply timeline.html]: Thanks *queen

File: portfolio.html [hunk 20]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>The Personal Portfolio of Me</title>
+
```
Comment (Roomsycakes01) [inline portfolio.html]: inconsistent white space

File: portfolio.html [hunk 21]
```diff
@@ -0,0 +1,97 @@
...
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
```
Comment (Roomsycakes01) [inline portfolio.html]: inconsistent white space

File: portfolio.html [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
```
Comment (Roomsycakes01) [inline portfolio.html]: inconsistent white space

File: portfolio.html [hunk 23]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
```
Comment (Roomsycakes01) [inline portfolio.html:4]: come inconsistent spacing after tag conditions

File: portfolio.html [hunk 24]
```diff
@@ -0,0 +1,97 @@
...
+				<a href="https://github.com/EleanorRoberts" class="fa fa-github-square"></a>
+			</div>
+		</section>
+	</div>
+	<!-- footer -->
+	<footer>
+		<div>NovaInc. Copyright 2021</div>
+	</footer>
+</body>
+</html>
```
Comment (Roomsycakes01) [inline portfolio.html]: extra line after html

File: timeline.html [hunk 25]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
```
Comment (Roomsycakes01) [inline timeline.html:4]: inconsistent spacing after classes

File: timeline.html [hunk 26]
```diff
@@ -0,0 +1,80 @@
...
+    <title>Timeline</title>
+
```
Comment (Roomsycakes01) [inline timeline.html]: inconsistent white space

File: timeline.html [hunk 27]
```diff
@@ -0,0 +1,80 @@
...
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
```
Comment (Roomsycakes01) [inline timeline.html]: inconsistent white space

File: timeline.html [hunk 28]
```diff
@@ -0,0 +1,80 @@
...
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
```
Comment (Roomsycakes01) [inline timeline.html:15]: inconsistent white space

File: timeline.html (hunk 18 above)
Comment (Roomsycakes01) [inline timeline.html]: extra line at the bottom

File: portfoliostyle.css [hunk 29]
```diff
@@ -0,0 +1,438 @@
+
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: remove extra white space

File: portfoliostyle.css [hunk 30]
```diff
@@ -0,0 +1,438 @@
...
+
+* {
+	box-sizing: border-box;
+	font-family: 'Nunito', sans-serif;
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: Don't think this needs quotes as it's only 1 word

File: portfoliostyle.css (hunk 29 above)
Comment (jesshamby) [inline portfoliostyle.css]: Remove excess whitespace to condense code and keep it consistent :)

File: portfoliostyle.css [hunk 31]
```diff
@@ -0,0 +1,438 @@
...
+}
+
+h3 {
+	color: #000000;
+	border-radius: 15px;
+	padding: 5px 0;
+}
+
+p a {
+	color: #FFFFFF;
```
Comment (jesshamby) [inline portfoliostyle.css:76]: Lots of elements with color: FFFFFF; which you could perhaps combine into one and cut down the number of lines of code:)

File: portfoliostyle.css [hunk 32]
```diff
@@ -0,0 +1,438 @@
...
+}
+
+body {
+	background-color: #D3D3D3;
+	background-image: linear-gradient(#808080, #FFFFFF);
+	background-position: top center;
+}
+
+/*most element styles*/
+
```
Comment (Roomsycakes01) [inline portfoliostyle.css:17]: Having the nav background change colour as well as the links overloads the brain a bit

File: portfoliostyle.css [hunk 33]
```diff
@@ -0,0 +1,438 @@
...
+.container {
+	padding: 10px 40px;
+	position: relative;
+	width: 60%;
+}
+
+/* the balls on the timeline*/
+.container::after {
+	content: '';
```
Comment (jesshamby) [inline portfoliostyle.css]: Consistency with ' or " - seems you usually go for " :)

File: portfoliostyle.css [hunk 34]
```diff
@@ -0,0 +1,438 @@
...
+	align-items: flex-end;
+}
+
+/* most class styles */
+
+.to-hide {
+	display: none;
+}
+
+.project-info:hover .to-hide, .container:hover .to-hide {
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: would recommend trying to make this so it stays unhidden after a click, also it isn't keyboard navigable so maybe add a :focus to it.

File: portfoliostyle.css [hunk 35]
```diff
@@ -0,0 +1,438 @@
+
+
+* {
```
Comment (Roomsycakes01) [inline portfoliostyle.css:1]: inconsistent spacing after *

File: portfoliostyle.css [hunk 36]
```diff
@@ -0,0 +1,438 @@
...
+	height: 0;
+	position: absolute;
+	top: 22px;
+	width: 0;
+	z-index: -1;
+	left: 15px;
+	border: medium solid #FFFFFF;
+	border-width: 8px 25px 8px 0;
+	border-color: transparent #808080
+ transparent transparent;
```
Comment (jesshamby) [inline portfoliostyle.css]: Maybe keep this on the line above so it is clearer :)

File: timeline.html [hunk 37]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Timeline</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (jesshamby) [inline timeline.html:7]: Check consistency with indentation in the head tag :)

File: portfoliostyle.css [hunk 38]
```diff
@@ -0,0 +1,438 @@
...
+}
+
+.left {
+	left: -10%;
+}
+
+.right {
+	left: 50%;
+}
+
```
Comment (Roomsycakes01) [inline portfoliostyle.css:259]: maybe add a comment to show this css formats the left triangle

File: timeline.html [hunk 39]
```diff
@@ -0,0 +1,80 @@
...
+	<nav>
+		<a href="portfolio.html" class="nav-item">Home</a>
+		<a href="eleanor-roberts-cv.pdf" class="nav-item cv" target="_blank">CV</a>
+	</nav>
+	<!-- header -->
+	<header id="home">
+		<h1>History of Me</h1>
+	</header>
+	<!-- timeline -->
+	<section id="timeline">
```
Comment (jesshamby) [inline timeline.html]: Do you need this section and/or the first div child? Could you combine the style properties :)

Comment (jesshamby) [reply timeline.html]: Ahh woops see I've commented on this before so you must need it :)

File: portfoliostyle.css [hunk 40]
```diff
@@ -0,0 +1,438 @@
...
+		border: 0;
+		margin: 0 auto;
+		flex-direction: row;
+		justify-content: space-evenly;
+		align-items: center;
+		flex-wrap: wrap;
+		background-color: transparent;
+	}
+	
+	nav:hover {
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: I'm not sure this is doing anything

**Review by jesshamby — COMMENTED** [review]
Looking good - just a few changes :)

File: portfoliostyle.css [hunk 41]
```diff
@@ -0,0 +1,438 @@
...
+	header {
+		height: auto;
+	}
+
+	h1 {
+		padding: 9vw;
+		font-size: 12vw;
+	}
+	
+    .timeline::after {
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: indentation out of line

File: portfoliostyle.css [hunk 42]
```diff
@@ -0,0 +1,438 @@
...
+
+	.media-links {
+		flex-wrap: wrap;
+	}
+
+	.media-links a {
+		flex-wrap: wrap;
+		padding: 10px;
+	}
+    .left, .right {
```
Comment (Roomsycakes01) [inline portfoliostyle.css]: indentation out of line

File: portfoliostyle.css [hunk 43]
```diff
@@ -0,0 +1,438 @@
...
+
+body {
+	background-color: #D3D3D3;
+	background-image: linear-gradient(#808080, #FFFFFF);
+	background-position: top center;
+}
+
+/*most element styles*/
+
+nav {
```
Comment (Roomsycakes01) [inline portfoliostyle.css:18]: maybe change the text in the nav to have more contrast for accessibility, especially after the media query.

File: portfolio.html (hunk 20 above)
Comment (EleanorRoberts) [reply portfolio.html]: I separated out the link tags I have copied from elsewhere to insert into my code (the three below)

File: portfolio.html (hunk 21 above)
Comment (EleanorRoberts) [reply portfolio.html]: see above

File: portfolio.html (hunk 23 above)
Comment (EleanorRoberts) [reply portfolio.html:4]: see comment above

File: portfoliostyle.css (hunk 29 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: 😅

File: portfoliostyle.css (hunk 33 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Removed all ' and replaced with " (excepting in content)

File: portfoliostyle.css (hunk 36 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Removed extra new line 😇

File: timeline.html (hunk 37 above)
Comment (EleanorRoberts) [reply timeline.html:7]: This was done purposefully to separate out the open source stylesheets, added a comment for clarity

File: timeline.html (hunk 39 above)
Comment (EleanorRoberts) [reply timeline.html]: Actually I think I missed it! I have now removed the extra unnecessary div 😅

File: portfolio.html (hunk 24 above)
Comment (EleanorRoberts) [reply portfolio.html]: This was added yesterday?

File: timeline.html (hunk 26 above)
Comment (EleanorRoberts) [reply timeline.html]: See above

File: timeline.html (hunk 28 above)
Comment (EleanorRoberts) [reply timeline.html:15]: See above

File: timeline.html (hunk 18 above)
Comment (EleanorRoberts) [reply timeline.html]: See above

File: portfoliostyle.css (hunk 29 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Done

File: portfoliostyle.css (hunk 30 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: I would prefer to keep the quotes in for consistency if I add fonts which have two words?

File: portfoliostyle.css (hunk 32 above)
Comment (EleanorRoberts) [reply portfoliostyle.css:17]: Is this personal preference or in relation to flashing?

File: portfoliostyle.css (hunk 35 above)
Comment (EleanorRoberts) [reply portfoliostyle.css:1]: ?

File: portfoliostyle.css (hunk 38 above)
Comment (EleanorRoberts) [reply portfoliostyle.css:259]: .... It doesn't it formats the container 🤔

File: portfoliostyle.css (hunk 40 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Removed ^^

File: portfoliostyle.css (hunk 42 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Fixed

File: portfoliostyle.css (hunk 41 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Fixed

File: portfoliostyle.css (hunk 32 above)
Comment (EleanorRoberts) [reply portfoliostyle.css:17]: Amended to not change with hover

File: portfoliostyle.css (hunk 34 above)
Comment (EleanorRoberts) [reply portfoliostyle.css]: Added focus to elements which unhides the p tag

File: portfoliostyle.css (hunk 43 above)
Comment (EleanorRoberts) [reply portfoliostyle.css:18]: Changed to have better contrast

**Comment by mclack** [comment]
- Your CSS file contains a bit of redundant code here and there that could do with being removed as it's not having an effect on your page.

- It may also be an idea to add a target attribute to your media-links and project <a> tags with a value of "_blank". This will open the links in a new tab rather than directing away from the current page.

Apart from that it's looking really tidy and professional! Good work.

**Pull request merged**